internal/kibana/security_exception_list/schema.go

```diff
@@ -0,0 +1,126 @@
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+//go:embed resource-description.md
+var exceptionListResourceDescription string
+
+func (r *ExceptionListResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: exceptionListResourceDescription,
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				MarkdownDescription: "The unique identifier of the exception list (auto-generated by Kibana).",
+				Computed:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"space_id": schema.StringAttribute{
+				MarkdownDescription: "An identifier for the space. If space_id is not provided, the default space is used.",
+				Optional:            true,
+				Computed:            true,
+				Default:             stringdefault.StaticString("default"),
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"list_id": schema.StringAttribute{
+				MarkdownDescription: "The exception list's human readable string identifier.",
+				Required:            true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"name": schema.StringAttribute{
+				MarkdownDescription: "The name of the exception list.",
+				Required:            true,
+			},
+			"description": schema.StringAttribute{
+				MarkdownDescription: "Describes the exception list.",
+				Required:            true,
+			},
+			"type": schema.StringAttribute{
+				MarkdownDescription: "The type of exception list. Can be one of: `detection`, `endpoint`, `endpoint_trusted_apps`, `endpoint_events`, `endpoint_host_isolation_exceptions`, `endpoint_blocklists`.",
+				Required:            true,
+				Validators: []validator.String{
+					stringvalidator.OneOf(
+						"detection",
+						"endpoint",
+						"endpoint_trusted_apps",
+						"endpoint_events",
+						"endpoint_host_isolation_exceptions",
+						"endpoint_blocklists",
+					),
+				},
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"namespace_type": schema.StringAttribute{
+				MarkdownDescription: "Determines whether the exception list is available in all Kibana spaces or just the space in which it is created. Can be `single` (default) or `agnostic`.",
+				Optional:            true,
+				Computed:            true,
+				Default:             stringdefault.StaticString("single"),
+				Validators: []validator.String{
+					stringvalidator.OneOf("single", "agnostic"),
+				},
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"os_types": schema.ListAttribute{
```

Comment:
```suggestion
			"os_types": schema.SetAttribute{
```

Similarly I suspect this should be a set

Reply:
https://github.com/elastic/terraform-provider-elasticstack/pull/1495/commits/5e940b33f4e5ce55ca56ecc5ca0a6df1c098e12e